src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThrough8.java

```diff
@@ -14,15 +14,15 @@ void methodLastLine(int i) {
             switch (i) {
                 case 0:
                     i++;
-                    /* block */ // comment
-                    // fall thru
+                 /* block */ // comment
+                 // fall thru
                 case 1:
                     i++;
                     break;
                 case 2:
                     i++;
-                    /* comment */ /* fall thru */ /* comment */
-                case 3: // violation 'Fall\ through from previous branch of the switch statement'
+                 /* block */ /* fallthru */ // comment
```

Comment:
Restore as it was before.

Reply:
@romani but this violation should be removed because of fall thru comment above

Reply:
Violation line, line 25 should be changed, all other lines should not

Reply:
Done @romani